blender/LilySurfaceScraper/Scrapers/PolyHavenHdriScraper.py

```diff
@@ -71,9 +89,9 @@ def fetchVariant(self, variant_index, material_data):
             return False
         
         var_name = variants[variant_index]
-        material_data.name = "polyhaven/" + identifier + '/' + var_name
+        material_data.name = f"{self.home_dir}/{identifier}/{var_name}"
 
-        map_url = variant_data[var_name]['exr']['url']
+        map_url = variant_data[var_name]['hdr']['url']
```

Comment:
I prefere EXRs, if you want HDRs to be available we will add it to the variant list (like for textures).